network-api/networkapi/wagtailpages/translation.py

```diff
@@ -5,7 +5,6 @@
     NewsPage,
     InitiativesPage,
     ParticipatePage2,
-    PeoplePage,
```

Comment:
let's keep this, but move it to the end of the list here, with a short `# obsolete imports` right above it (so that we know anything after that comment is obsolete and requires cleanup)